cmd/minishift/cmd/delete.go

```diff
@@ -36,6 +37,10 @@ var deleteCmd = &cobra.Command{
 }
 
 func runDelete(cmd *cobra.Command, args []string) {
+	instancedata.Data.ReadInstanceData()
+
+	fmt.Println("ocPath in delete ", instancedata.Data.OcPath)
```

Comment:
Why would you add this? I thought you just had this for testing purposes. Do you really want this to be part of the final pull request? What do you try to communicate to the user here?